xml/article_pacemaker_remote.xml

```diff
@@ -405,9 +405,9 @@ ssh: connect to host &node3; port 3121: Connection refused</screen>
    <para>To integrate the remote node into the cluster, proceed as follows:</para>
    <procedure>
     <step>
-     <para>Log in to each cluster node and make sure &pace; service is already
-      started:</para>
-     <screen>&prompt.root;<command>crm</command> cluster start</screen>
+     <para>Make sure the &pace; service is already started on all of the cluster
```

Comment:
This sounds not make sense.
If cluster service is already started, then there is no need to run `crm cluster start` or `crm cluster start --all`;
`crm cluster start` and `crm cluster start --all` are the commands to start cluster service.

Maybe the more accurate way is:
```
Make sure all of the cluster nodes already configured successfully, and already stopped
```

Reply:
I think the idea here was that if the services are stopped, this command will start them; and if they're already started, it just gives you an INFO output saying the cluster services are already started. So either way you end up with started services. 

Would it make more sense to say something like: 

> Make sure the cluster services are already started on all of the cluster nodes. If not, run the following command to start the cluster services on all nodes:
> `crm cluster start --all`